lib/launchers/openstack.rb

```diff
@@ -28,6 +28,11 @@ def self.instance(**options)
                      'openstack_qeos10'
       opts = default_opts(service_name).merge options
 
+      # for private OSP behind proxy
+      if opts[:proxy]
+        ENV['http_proxy'] = opts[:proxy]
```

Comment:
Due to https://github.com/openshift/verification-tests/blob/master/lib/http.rb#L44

Reply:
I had to revert because it got merged before I finished my comment. I'm wondering why do we need an OSP behind proxy support. It is fine, just curious why we use such setup.

I think we should pass the `proxy` option to `Http` instead of altering the env variable.